river/metrics/preq_roc_auc.py

```diff
@@ -0,0 +1,67 @@
+from river import metrics, utils
+
+from .efficient_preqrocauc import EfficientPreqROCAUC
+
+__all__ = ["PreqROCAUC"]
+
+
+class PreqROCAUC(metrics.base.BinaryMetric):
```

Comment:
Could we call this `RollingROCAUC` instead? The rolling terminology is aligned with the rest of River.

Reply:
Renamed.